go.mod

```diff
@@ -1,27 +1,32 @@
 module github.com/peggyjv/gravity-bridge
 
-go 1.18
+go 1.19
 
 require (
-	github.com/cosmos/cosmos-sdk v0.46.7
+	cosmossdk.io/simapp v0.0.0-20221216140705-ee8890cf30e7
+	github.com/cosmos/cosmos-sdk v0.47.0-rc1
```

Comment:
there's a newer release candidate that uses CometBFT: https://github.com/cosmos/cosmos-sdk/releases/tag/v0.47.0-rc3

Reply:
was planning to do the upgrade with the migration to cometbft in another PR since that one has been opened for quite a long time. 

I will do it in a separate branch on top of this one